### PR TITLE
My Portfolio Project Desktop Version

### PR DESCRIPTION
**Pull request by CaptainBawa**
Add NavBar and Hero Section.
Add Work and About Section.
Add Form and Footer Section.
Styled with Grid and Flexbox.

### CODE REVIEW

File: desktop.css [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+  /* work section */
+
+  .main-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 130px 24px;
+    width: 100%;
+    height: auto;
+  }
```
Comment (leonardodiasb) [inline desktop.css]: - [ ] You did a great job implementing the hover effect on the regular project cards, but the projects section is not responsive and on some small desktop screen sizes, the page cuts the elements. Kindly make it responsive so all the cards can appear on the screen even on small desktop sections.
 - Your design:
 - 
image

 - Figma template:
 - 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @CaptainBawa, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Nice hover effects. ✔️ 
 - Correct use of GitHub flow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: desktop.css [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+    display: flex;
+    align-items: center;
+    color: #091e42;
+  }
+
+  .vector {
+    width: 100%;
+  }
+
+  .work-cards {
```
Comment (huseyinbiyik) [inline desktop.css:195]: Your design implementation looks good 👍  But unfortunately, there is a new bug introduced while fixing the previous reviewer's request. Now some cards are beyond the view of the left side of the screen as you can see in the GIF below. 
[OPTIONAL] Unfortunately, the previous reviewer didn't notice that and I need to keep it optional but, not only works/projects section, almost all sections have responsive issues. Kindly fix them to have a good-looking responsive portfolio 👍 
Screen Recording 2023-03-02 at 9 29 31 PM

File: desktop.css [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+    width: 100%;
+    height: 72px;
+    background-color: #3e3c3c;
+  }
+
+  .desktop-navbar {
+    display: flex;
+    gap: 8px;
+    padding: 1.5% 18% 0 10%;
+  }
```
Comment (huseyinbiyik) [inline desktop.css]: [OPTIONAL] Kindly make sure your navigation is aligned center and the elements between the nav link has same space between each other like on the Figma template.
Screenshot 2023-03-02 at 9 43 09 PM
Screenshot 2023-03-02 at 9 45 19 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @CaptainBawa,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝


 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi @CaptainBawa  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**